spk/ntopng/src/conf/privilege

```diff
@@ -0,0 +1,31 @@
+{
+	"defaults":{
+		"run-as": "package"
+	},
+	"username": "sc-ntopng",
```

Comment:
I would set defaults to "run-as : root" and probably discard "username" if there is no interest to have such a package specific user either for service execution or package file ownership.

Reply:
Only the service start needs to be done as root. So as root the ownership of some files (pid, ...) is set to sc-ntopng and the service itself is run as this package specific user.
So we definitifly need the package user.
And the package needs libcap (that makes this possible as far as I understand)
[Libcap implements the user-space interfaces to the POSIX 1003.1e capabilities available in Linux kernels. These capabilities are a partitioning of the all powerful root privilege into a set of distinct privileges. ]